tests/Moq.Tests/VerifyFixture.cs

```diff
@@ -6,6 +6,7 @@
 using Xunit;
 
 using System.Threading.Tasks;
+using Moq.Protected;
```

Comment:
Nitpick: Please add one blank line above this one. In the Moq codebase, `using`s are generally grouped into blocks by their main namespace. `System.*` comes first, then any other groups follow in alphabetical order.

(`Xunit` should actually be placed after `System.*` but since that was already wrong before your PR, we'll just leave that as is for the moment.)

Reply:
Done